lib/phantom-script.js

```diff
@@ -69,8 +69,8 @@ page.open(skeleton, function (status) {
   page.paperSize = defaultFormat;
 
   if (args.runningsPath !== "nofile") {
-    var runningsArgs = JSON.parse(args.runningsArgs);
-    var runnings = require(args.runningsPath)(runningsArgs);
+    var runningsArgs = args.runningsArgs ? JSON.parse(args.runningsArgs) : undefined;
```

Comment:
Unless I'm misunderstanding you, in which case, I'm happy to do re-write the code. 

Try running this following code in the console, to see what I mean:
```javascript
var str = '';
console.log(str ? 'truthy' : 'falsey');
```